pages/linux/cal.md

```diff
@@ -6,13 +6,13 @@
 
 `cal`
 
-- Use monday as the first day of the week:
+- Display previous, current and next month:
 
-`cal -m`
+`cal --three`
 
-- Display a calendar for the current year:
+- Use monday as the first day of the week:
 
-`cal -y`
+`cal --monday`
```

Comment:
Please wrap monday in tokens ({{}})

Reply:
Sorry, but I'm not sure I understood what you meant.
I got the impression that you want this: `cal {{--monday}}`
Is this right ?

Reply:
cal --{{monday}}. https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax

Reply:
This wouldn't cause confusion as someone could think that `{{monday}}` could be swapped for any other day ?

Reply:
Oh I see that only monday or sunday is applicable. I initially thought any day can be used.